packages/quill/src/core/quill.ts

```diff
@@ -193,22 +194,45 @@ class Quill {
   options: ExpandedQuillOptions;
 
   constructor(container: HTMLElement | string, options: QuillOptions = {}) {
-    this.options = expandConfig(container, options);
-    this.container = this.options.container;
-    if (this.container == null) {
-      debug.error('Invalid Quill container', container);
-      return;
+    this.setup(container, options);
+  }
+
+  configure(options: QuillOptions = {}) {
```

Comment:
Since this is a public method, maybe we could add a JSDoc for it as well as mention it in the documentation?